src/main/java/com/solvd/movie/kafka/parser/XmlParser.java

```diff
@@ -0,0 +1,23 @@
+package com.solvd.movie.kafka.parser;
+
+import com.jcabi.xml.XML;
+import com.jcabi.xml.XMLDocument;
+import lombok.SneakyThrows;
+
+import java.util.Objects;
+
+public class XmlParser {
```

Comment:
this is duplicate of https://github.com/hizmailovich/ReviewMicroservice/pull/10/files/74ecfdc83fd41780382a3456208580f5ee094d5e#diff-d6274374791ec83b8e9f4801dd751f779edeed900b71b9a6962bd8a47d201a9a

Reply:
Fixed, I made this class abstract

Reply:
its still a copy of that class